css/style.css

```diff
@@ -0,0 +1,411 @@
+*,*::after,*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 14px;
+  line-height: 1.42857143;
+  color: #333;
+  background-color: #fff;
+}
+
+div, article, section, header, footer {
+  display: block;
+}
+
+h1, h2, h3, h4 {
+  font-family: 'Roboto', sans-serif;
+  line-height: 1.2;
+  margin-bottom: 20px;
+  color: #2F362F;
+}
+
+h2 {
+  display: block;
+  margin-block-start: 0.83em;
+  margin-block-end: 0.83em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  font-size: 34px;
+}
+
+h3 {
+  display: block;
+  font-size: 1.17em;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  font-weight: 500;
+}
+
+h4 {
+  display: block;
+  margin-block-start: 1.33em;
+  margin-block-end: 1.33em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  font-weight: bold;
+}
+
+p {
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  color: #7c8595;
+  margin: 0 0 10px;
+  margin-bottom: 15px;
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+}
+
+a {
+  color: #7c8595;
+  text-decoration: none;
+  background-color: transparent;
+}
+
+img {
+  vertical-align: middle;
+  border: 0;
+}
+
+.width-100 {
+  width: 100%;
+}
+
+.container {
+  padding-right: 15px;
+  padding-left: 15px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.row {
+  margin-right: -15px;
+  margin-left: -15px;
+}
+
+.margin-bottom-40 {
+  padding-bottom: 40px;
+}
+
+.margin-bottom-30 {
+  padding-bottom: 30px;
+}
+
+.padding-bottom-120 {
+  padding-bottom: 127px;
+}
+
+.columns {
+  position: relative;
+  min-height: 1px;
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+.section-white {
+  background-color: #fff;
+  border-top: 1px solid #eaeaea;
+  padding: 70px 0 60px 0;
+}
+
+.title-section {
+font-size: 36px;
+line-height: 42px;
+margin: 0 0 10px 0;
+}
+
+.subtitle-section{
+  font-size: 18px;
+}
+
+.align-all-center {
+  justify-content: center;
+  align-items: center;
+}
+
+.flex {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.subtitle {
+  display: block;
+  padding: 20px 0;
+  background-color: #f5f6f7;
+  text-decoration: none;
+  color: #3b3939;
+  font-size: 18px;
+  transition: all 0.3s linear;
+}
+
+.button {
+  color: #373737;
+  font-size: 17px;
+  font-weight: 500;
+  background-color: #fff;
+  box-shadow: 0px 8px 40px 0 rgba(0, 0, 0, 0.37);
+  letter-spacing: 1px;
+  border-radius: 5px 5px;
+  margin: 10px 10px 0 10px 0;
+  padding: 18px 30px;
+  display: inline-block;
+  transition: all 0.4s ease-in-out;
+}
+
+.start-creating-and-header-button:hover {
```

Comment:
Entendi.. porém seria melhor descrever o comportamento no nome da classe quando é assim.

Se essa shadow for reutilizada em todos os ctas do site poderia ser `cta-default-shadow`